### PR TITLE
Feature: Popups

### PR DESCRIPTION
**Pull request by luftedar**
In this update: 

- Add Popup Windows for Projects for mobile and desktop version.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -16,4 +16,140 @@ for (let i = 0; i < mobileMenuItems.length; i += 1) {
 const closeMenu = document.querySelector('#toggle-x');
 closeMenu.addEventListener('click', () => {
   menu.classList.add('display-none');
-});
\ No newline at end of file
+});
+
+const projectsArr = [
```
Comment (SajjadAhmad14) [inline script.js:21]: Please add the `X` sign to the pop detail page for mobile. Kindly see the screenshot below for reference.
cross

File: index.html [hunk 2]
```diff
@@ -196,6 +196,68 @@ <h3>Multi-Post Stories<br />Gain+Glory</h3>
...
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+            Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem 
+            Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's 
+            standard dummy text ever since the 1500s, 
+            when an unknown printer took a galley of type and scrambled it 1960s with the relea
+          </p>
+          <div id="mobile-popup-buttons" class="d-flex">
+            <a href="#" class="pop-up-button g-btn" id="see-live">See Live <img class="popicons" src="img/PopupLive.png" alt=""></a>
+            <a href="#" class="pop-up-button g-btn" id="see-source">See Source <img class="popicons" src="img/PopupGit.png" alt=""></a>
```
Comment (SajjadAhmad14) [inline index.html:226]: Please add links for the live version and source code.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Orcun Ugur, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html:226]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Orcun, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer(s). There is nothing else to say other than... it's time to merge it 🍾 :shipit:.
Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**